csrankings.csv

```diff
@@ -17046,10 +17046,10 @@ eva Kühn,TU Wien,http://www.complang.tuwien.ac.at/eva,NOSCHOLARPAGE
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap,v2WiJeIAAAAJ
 Øystein Nytrø,NTNU,https://www.ntnu.edu/employees/nytroe,-Zavs80AAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
-Chen Hajaj,Ariel University,https://www.google.com/url?sa=t&source=web&cd=1&cad=rja&uact=8&ved=2ahUKEwjFgdSqhrniAhXN1qQKHUVdDzwQFjAAegQIARAB&url=https%3A%2F%2Fwww.ariel.ac.il%2Fwp%2Fchen-hajaj%2F&usg=AOvVaw01oGV2byT9wxOMYjET6x9O,
-Shani Alkoby,Ariel University,https://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=687&id_lang=2,
-Erel Segal-Halevi,Ariel University,http://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=614&id_lang=1,
-Gil Ben Arzi,Ariel University,http://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=645&id_lang=1,
-Tal Grinshoun,Ariel University,https://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=426&id_lang=2,
-Lihi Deri,Ariel University,https://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=489&id_lang=2,
-Inon Zukerman,Ariel University,https://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=316&id_lang=2,
\ No newline at end of file
+Chen Hajaj,Ariel University,https://www.google.com/url?sa=t&source=web&cd=1&cad=rja&uact=8&ved=2ahUKEwjFgdSqhrniAhXN1qQKHUVdDzwQFjAAegQIARAB&url=https%3A%2F%2Fwww.ariel.ac.il%2Fwp%2Fchen-hajaj%2F&usg=AOvVaw01oGV2byT9wxOMYjET6x9O,Zy2cIskAAAAJ

```

Comment:
Please fix this home page: it should not be a google link.

Reply:
My bad, it should be
https://www.ariel.ac.il/wp/chen-hajaj/

Updated the file